### PR TITLE
Update sunbird_discussions-mw-service.env

### PR DESCRIPTION
**Pull request by arunpilli21**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Enhancement (additive changes to improve performance)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env [hunk 1]
```diff
@@ -11,4 +11,17 @@ authorization_token={{ discussionsmw_nodebb_authorization_token }}
...
-API_AUTH_TOKEN={{ api_auth_token | default('') }}
\ No newline at end of file
+API_AUTH_TOKEN={{ api_auth_token | default('') }}
+moderation_flag={{ moderation_flag |  default(false) }}
+moderation_type={{ moderation_type |  default('pre-moderation') }}
+moderation_id={{ moderation_id |  default('GEbX4X0B9pbA_yqYBUtM') }}
+moderation_flag_by={{ moderation_flag_by |  default('system_flagged') }}
+moderation_platform={{ moderation_platform |  default('Sunbird') }}
+moderation_category={{ moderation_category |  default('discussions') }}
+moderation_contentId={{ moderation_contentId |  default('1112223332552') }}
```
Comment (VenkatKadiveti) [inline ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env:21]: moderation_contentId={{ moderation_contentId | default('') }}

File: ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env [hunk 2]
```diff
@@ -11,4 +11,17 @@ authorization_token={{ discussionsmw_nodebb_authorization_token }}
...
+moderation_email={{ moderation_email |  default('test@tarento.com') }}
```
Comment (VenkatKadiveti) [inline ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env:22]: moderation_email={{ moderation_email | default('') }}

File: ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env [hunk 3]
```diff
@@ -11,4 +11,17 @@ authorization_token={{ discussionsmw_nodebb_authorization_token }}
...
+moderation_email_subject={{ moderation_email_subject |  default('Sunbird Discussion  Moderation') }}
+moderation_course={{ moderation_course |  default('Sunbird training') }}
+LEARNER_SERVICE_URL={{ learner_service_url |  default('https://igot-dev.in') }}
```
Comment (VenkatKadiveti) [inline ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env:25]: LEARNER_SERVICE_URL={{ learner_service_url | default('') }}

File: ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env [hunk 4]
```diff
@@ -11,4 +11,17 @@ authorization_token={{ discussionsmw_nodebb_authorization_token }}
...
+SB_API_KEY={{ SB_API_KEY |  default('bearer abc') }}
```
Comment (VenkatKadiveti) [inline ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env:26]: SB_API_KEY={{ SB_API_KEY | default('') }}

**Review by VenkatKadiveti — APPROVED** [review]

File: ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env [hunk 5]
```diff
@@ -11,4 +11,17 @@ authorization_token={{ discussionsmw_nodebb_authorization_token }}
 TELEMETRY_EVENTS_BATCH_SIZE={{ telemetry_events_batch_size | default(1) }}
 TELEMETRY_SERVICE_API_SLUG={{ telemetry_service_api_slug | default('/v1/telemetry') }}
 TELEMETRY_SERVICE_URL={{ telemetry_service_url | default('http://telemetry-service:9001') }}
-API_AUTH_TOKEN={{ api_auth_token | default('') }}
\ No newline at end of file
+API_AUTH_TOKEN={{ api_auth_token | default('') }}
+moderation_flag={{ moderation_flag |  default(false) }}
+moderation_type={{ moderation_type |  default('pre-moderation') }}
+moderation_id={{ moderation_id |  default('GEbX4X0B9pbA_yqYBUtM') }}
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env:17]: What is this id? Is this a constant across all environments?

File: ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env [hunk 6]
```diff
@@ -11,4 +11,17 @@ authorization_token={{ discussionsmw_nodebb_authorization_token }}
...
+moderation_flag_by={{ moderation_flag_by |  default('system_flagged') }}
+moderation_platform={{ moderation_platform |  default('Sunbird') }}
+moderation_category={{ moderation_category |  default('discussions') }}
+moderation_contentId={{ moderation_contentId |  default('') }}
+moderation_email={{ moderation_email |  default('') }}
+moderation_email_subject={{ moderation_email_subject |  default('Sunbird Discussion  Moderation') }}
+moderation_course={{ moderation_course |  default('Sunbird training') }}
+LEARNER_SERVICE_URL={{ learner_service_url |  default('') }}
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env:25]: Why is this empty? Can we use the service url as default?

File: ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env [hunk 7]
```diff
@@ -11,4 +11,17 @@ authorization_token={{ discussionsmw_nodebb_authorization_token }}
...
+SB_API_KEY={{ SB_API_KEY |  default('') }}
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env:26]: Which all APIs access is required for this change? We may need to generate the key, so I would like to know the list of apis required.

File: ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env [hunk 8]
```diff
@@ -11,4 +11,17 @@ authorization_token={{ discussionsmw_nodebb_authorization_token }}
 TELEMETRY_EVENTS_BATCH_SIZE={{ telemetry_events_batch_size | default(1) }}
 TELEMETRY_SERVICE_API_SLUG={{ telemetry_service_api_slug | default('/v1/telemetry') }}
 TELEMETRY_SERVICE_URL={{ telemetry_service_url | default('http://telemetry-service:9001') }}
-API_AUTH_TOKEN={{ api_auth_token | default('') }}
\ No newline at end of file
+API_AUTH_TOKEN={{ api_auth_token | default('') }}
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env:14]: What is the difference between this variable and `SB_API_KEY`?

**Review by keshavprasadms — APPROVED** [review]

**Pull request merged**